frontend/src/hooks/useAuth.js

```diff
@@ -94,11 +94,13 @@ function useProvideAuth() {
    * @param {string} token
    */
   const tokenLogin = (token) => {
-    if (!authRequestPending() && !store.getState().appHandler.isLogged) {
+    if (!authRequestPending() && !isLogged) {
```

Comment:
@siemiatj  Q: instead of using `isLogged` which is calculated when `useProvideAuth()` is called,
isn't it better to compute the `isLogged` inside the `tokenLogin` function, so we will have the fresh value of it? (Same thing for the other functions too)

What's the reason behind this?